src/Utilities/SplashHandler.cs

```diff
@@ -0,0 +1,90 @@
+//-----------------------------------------------------------------------------
+// <copyright file="SplashHandler.cs" company="WheelMUD Development Team">
+//   Copyright (c) WheelMUD Development Team.  See LICENSE.txt.  This file is 
+//   subject to the Microsoft Public License.  All other rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using System;
+
+namespace WheelMUD.Utilities
+{
+    public static class SplashHandler
```

Comment:
Although this is in Core, a game implementation would very likely end up replacing or customizing this code directly.  Leaving them in a state where they have mergeability divergence from our base repo.  (This is a topic I need to discuss more about in some guiding development principles document or something; taking a note locally.  It would be foolish to think we can 100% avoid games needing to customize Core code now and then, but we'd like to minimize the need for those who want a healthy two-way merge path with base repo.  Modern code sharing strategies are something I hope will accelerate the rate of advancement of the MUDs of the future.)

Recommend we track a ticket to move this code into the Renderer pattern, where Core exports from a `DefaultSplashScreenRenderer.cs` but an individual game need only export a splash screen renderer with a higher priority to replace it.

As an added bonus, I imagine this would give _another_ novel way to make a temporary splash screen (like for a holiday or april first) by dropping a tiny .dll into the game folder (that includes the holiday splash renderer with a yet higher priority) for the non-reboot updates system to pick up, and delete the .dll when ready to get rid of it for another non-reboot update.  Or they could just modify their game-exported version and change it back when ready too.